src/server/models/url.ts

```diff
@@ -86,6 +88,19 @@ export const Url = <UrlTypeStatic>sequelize.define(
       type: Sequelize.BOOLEAN,
       allowNull: false,
     },
+    contactEmail: {
+      type: Sequelize.TEXT,
+      allowNull: false,
+      validate: {
+        isEmail: true,
+        isLowercase: true,
+        is: emailValidator.makeRe(),
+      },
```

Comment:
isn't there an inconsistency, if the default value populated by the SQL migration is an empty string, but here we are insisting on `allowNull: false` and `isEmail: true`?

would it be easier to simply allow null values?